SpotifyAPI.Web/SpotifyWebClient.cs

```diff
@@ -17,6 +17,13 @@ internal class SpotifyWebClient : IClient
         public ProxyConfig ProxyConfig { get; set; }
 
         private readonly Encoding _encoding = Encoding.UTF8;
+        private readonly HttpClient _client;
+
+        public SpotifyWebClient()
+        {
+            HttpClientHandler clientHandler = CreateClientHandler(ProxyConfig);
```

Comment:
This line won't work, since `ProxyConfig` will always be null. I guess adding it to the constructor with default to null would work...? It also doesnt make sense keeping it as a property, does it?